plugin/src/index.ts

```diff
@@ -94,15 +94,15 @@ export async function onPostBuild({
 
 export async function onSuccess() {
   // Pre-warm the lambdas as downloading the datastore file can take a while
-  if (shouldSkipBundlingDatastore()) {
+  if (shouldSkipBundlingDatastore() && process.env.DEPLOY_PRIME_URL) {
```

Comment:
[sand] the DEPLOY_PRIME_URL check could potentially be abstracted into `shouldSkipBundlingDatastore` since that also performs an env check